polkadot/runtime/parachains/src/runtime_api_impl/v10.rs

```diff
@@ -451,7 +451,16 @@ pub fn backing_state<T: initializer::Config>(
 	//
 	// Thus, minimum relay parent is ensured to have asynchronous backing enabled.
 	let now = frame_system::Pallet::<T>::block_number();
-	let min_relay_parent_number = shared::AllowedRelayParents::<T>::get()
+
+	// Workaround for issue #64.
```

Comment:
I think you meant to add this to https://github.com/paritytech/polkadot-sdk/pull/5423

Reply:
Yeah, and this explains why I messed up in https://github.com/paritytech/polkadot-sdk/pull/5423#discussion_r1781046770